packages/babel-parser/src/util/whitespace.js

```diff
@@ -21,6 +21,12 @@ export function isNewLine(code: number): boolean {
   }
 }
 
+// Skip whitespace and single-line comments including /* no newline here */.
+// After this RegExp matches, its lastIndex points either to line terminator
+// or start of multi-line comment.
+export const skipToLineBreak =
+  /(?:[^\S\n\r\u2028\u2029]|\/\/.*|\/\*.*?\*\/)*(?=[\n\r\u2028\u2029]|\/\*)/y;
```

Comment:
For reviewers: `[^\S\n\r\u2028\u2029]` is basically set subtraction: `\s - [\n\r\u2028\u2029]`.

I will probably write a plugin for https://github.com/tc39/proposal-regexp-set-notation#ecmascript-proposal-regexp-set-notation--properties-of-strings so that we can make it easier to read :grimacing:

Reply:
@lightmare Is it ok that it matches the first comment in this code?
```js
/* abc *//* def */ b
a
```

Also, I would prefer to call it `skipWhiteSpaceToLineBreak`.

Reply:
That's a bug, it shouldn't match. I'll rename it along with a fix.